templates/config.tfvars.aws-template

```diff
@@ -49,3 +49,4 @@ provision = {
   }
 }
 
+provisioner_image = "provisioner-image-name-autogenerated"
```

Comment:
Keep in config whar provisioner-image was used for the init of the config directory